Gopkg.toml

```diff
@@ -37,7 +37,9 @@
   version = "~1.3.0"
 
 [[constraint]]
-  version = "~0.1.0"
+  #version = "~0.1.0"
+  branch = "bug-1576728"
+  source = "github.com/shawn-hurley/bundle-lib"
```

Comment:
are we doing this?

Reply:
No, this depends on this https://github.com/automationbroker/bundle-lib/pull/80